### PR TITLE
Added 'plugins' option

### PR DESCRIPTION
**Pull request by nicolagusso**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [X] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — CHANGES_REQUESTED** [review]
Can you please add a unit test?

File: README.md [hunk 1]
```diff
@@ -92,5 +92,16 @@ fastify.route({
 })
 ```
 
+## Plugins
+You can also extend the functionalities of the ajv instance embedded in this validator by adding new ajv plugins.
```
Comment (climba03003) [inline README.md:96]: Please mention it is not using the latest `ajv` version. The users should install the corresponding version of plugin.
https://github.com/fastify/fastify-response-validation/df4e7c1aaa395e59c2d74d2299a6e610a223fb51/master/package.json#L7

**Review by climba03003 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Uzlopak) [reply README.md:96]: Well. nowadays we use ajv 8. So maybe this PR can be reviewed again

Comment (climba03003) [reply README.md:96]: Yes, this comment is resolved. But still lacks of unit test.

**Comment by mcollina** [comment]
I would also like to see a bump in the Ajv version

**Comment by Uzlopak** [comment]
I adapted the changes for my refactor

**Comment by Eomm** [comment]
Superseded by #75

**Pull request closed without merging**